src/graphql/error-map.ts

```diff
@@ -415,6 +415,7 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
     case "SafeWrapperError":
     case "InvalidFeeProbeStateError":
     case "InvalidPubKeyError":
+    case "SkipProbeForPubkeyError":
```

Comment:
this should have a custom message. Sending `Unknown error occurred` is not good for this case

Reply:
This is just to satisfy the type checker. This error never gets returned to the user right now. It's just used in a conditional to determine if to call `withoutRoute` or `withRoute` on the builder.